.gitignore

```diff
@@ -286,3 +286,5 @@ __pycache__/
 *.btm.cs
 *.odx.cs
 *.xsd.cs
+
+*Unity*.dll
```

Comment:
I think i would like to leave libraries in repository (for quick configuration on laptop etc.)

Reply:
k, I removed these lines